server/server.js

```diff
@@ -151,4 +152,23 @@ app.delete('/contents/:id', (req, res) => {
   });
 });
 
+/* ------------------------ UserRegister ------------------------*/
+
+app.post('/users/register', (req, res) => {
+  const formData = req.body;
+  if (formData.username == null || formData.username === '') {
+    res.status(422).json("Vous vous êtes mal enregistré");
+  } else {
+    connection.query('INSERT INTO user SET ?', formData, (err, results) => {
+      if (err) {
+        console.log(err);
+        res.status(500).send("Erreur lors de l'enregistrement");
+      } else {
+        res.status(201).send({...formData, id:results.insertId });
```

Comment:
remember to clean any private stuff like password when sending back user data, something like `{...formData, password: null, ... }`